### PR TITLE
fix(docs): book component

### PR DESCRIPTION
**Pull request by GuzmanPI**
Adds an href property to the component, so it redirects.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/scullyio/scully/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

Improves the book component in the docs app. It also changes the landing components so they redirect to a proper url.

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Other... Please describe:

## What is the current behavior?

The book components in the landing page do not redirect.

Issue Number: #710 

## What is the new behavior?

The book components in the landing page redirect.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: apps/scully-docs/src/app/landing/landing.component.html [hunk 1]
```diff
@@ -109,20 +109,22 @@ <h3>TOOLS & RESOURCES</h3>
...
+      [href]="'docs/blog/'"
       [header]="'Create a Blog'"
       [text]="
         'Easy step by step to create your own blog in less than 5 minutes.'
       "
     >
     </app-book>
   </div>
+  <div class="guide-2">
+    <app-book
```
Comment (aaronfrost) [inline apps/scully-docs/src/app/landing/landing.component.html:123]: Why did you move the Getting Started to second position? Most people click that one. Shouldn't it be first?

**Comment by SanderElias** [comment]
@GuzmanPI can you gix the MC

File: apps/scully-docs/src/app/components/book/book.component.html [hunk 2]
```diff
@@ -1,8 +1,8 @@
-<div>
+<a [href]="href">
   <img src="assets/book.png" alt="Book icon" />
 
   <div class="card-content">
-    <h3>{{ header }}</h3>
+    <h3>{{ header }}...</h3>
```
[the remark this answers is not in the document]
Comment (GuzmanPI) [reply apps/scully-docs/src/app/components/book/book.component.html:5]: Let me fix that

File: apps/scully-docs/src/app/components/book/book.component.html [hunk 3]
```diff
@@ -1,8 +1,8 @@
-<div>
+<a [href]="href">
```
Comment (SanderElias) [inline apps/scully-docs/src/app/components/book/book.component.html:1]: Why don't you use routerlink?

File: apps/scully-docs/src/app/components/book/book.component.ts [hunk 4]
```diff
@@ -8,6 +8,7 @@ import { Component, Input, OnInit } from '@angular/core';
 export class BookComponent implements OnInit {
   @Input() header: string;
   @Input() text: string;
+  @Input() href: string;
```
Comment (SanderElias) [inline apps/scully-docs/src/app/components/book/book.component.ts:11]: Reusing an existing and meaningfull attribute might become an issue later on.

File: apps/scully-docs/src/app/landing/landing.component.html [hunk 5]
```diff
@@ -109,15 +109,17 @@ <h3>TOOLS & RESOURCES</h3>
 </div>
 
 <div class="container-tools">
-  <div class="guide-1" [routerLink]="'/docs/getting-started'">
+  <div class="guide-1">
```
Comment (SanderElias) [inline apps/scully-docs/src/app/landing/landing.component.html:112]: perhaps exchange the div for an <a> instead of bolting on things?
